test/integration/authorization_test.go

```diff
@@ -1364,14 +1364,6 @@ func TestBrowserSafeAuthorizer(t *testing.T) {
 	proxyVerb := []string{"api", "v1", "proxy", "namespaces", "ns", "pods", "podX1:8080"}
 	proxySubresource := []string{"api", "v1", "namespaces", "ns", "pods", "podX1:8080", "proxy", "appEndPoint"}
 
-	isUnsafeErr := func(errProxy error) (matches bool) {
```

Comment:
I think we need to update the `NewBrowserSafeAuthorizer` to include extra information when it changes the verb and that change leads to a deny.  Otherwise the error message will say "you cannot proxy here" when it should say "you cannot unsafe proxy here".

Reply:
Yeah I looked into it briefly when I had to drop this code, I did not see an obvious way to do it.

Reply:
@simo5 https://github.com/simo5/origin/pull/18